scripts/hostapd/hostapd.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=hostapd IEEE 802.11 Access Point
-After=sys-subsystem-net-devices-wlan0.device
-BindsTo=sys-subsystem-net-devices-wlan0.device
-
-[Service]
-Type=forking
-PIDFile=/var/run/hostapd.pid
-ExecStart=/usr/sbin/hostapd -B /etc/hostapd/hostapd.conf -P /var/run/hostapd.pid
-ExecStartPost=/bin/sh /etc/hostapd/nat.sh
```

Comment:
This is needed to NAT LAN traffic to Hyperboria. If my phone is not running cjdns, it can still reach h. addresses when connected to this AP.